tests/common/platform/device_utils.py

```diff
@@ -738,7 +776,8 @@ def verify_required_events(duthost, event_counters, timing_data, verification_er
                                            format(observed_start_count, observed_end_count))
 
 
-def advanceboot_loganalyzer_factory(duthost, request, marker_postfix=None):
+@pytest.fixture()
+def advanceboot_loganalyzer_factory(duthost, request, ptfadapter, marker_postfix=None):
```

Comment:
I think that this change will break the testcase - you are changing the definition of the function here, but the callers are still the same (eg., are below links). The ptfadapter arg doesn't get a default value, and the callers don't provide it. So it will most likely fail.

https://github.com/sonic-net/sonic-mgmt/blob/fd080593f3077666945bae1d9ab17887eccb7799/tests/common/platform/device_utils.py#L937

https://github.com/sonic-net/sonic-mgmt/blob/fd080593f3077666945bae1d9ab17887eccb7799/tests/common/platform/device_utils.py#L964C1-L965C1

Reply:
If you agree then how is this change tested and why this was not caught in testing? Do we have a miss somewhere?